projects/elife.yaml

```diff
@@ -76,7 +76,8 @@ defaults:
             engine: postgres # or 'MySQL'
             # ensure this matches the version of Postgres you install on server!
             # version: '9.4' # EOL Feb 2020
-            version: '10.9' # EOL 2022
+            version: '11.1' # EOL Nov 2023
+            allow-major-version-upgrade: False # do not alter unless you're doing a major version upgrade
```

Comment:
the naming of this is consistent with RDS terminology, which is what we should use :+1: